src/aspire/utils/misc.py

```diff
@@ -187,6 +187,6 @@ def inverse_r(size, x0=0, y0=0, peak=1, dtype=np.float64):
     g = grid_2d(size, shifted=True, normalized=False, dtype=dtype)
 
     # Compute the denominator
-    circ = np.sqrt(1 + g["x"] ** 2 + g["y"] ** 2)
+    circ = np.sqrt(1 + (g["x"] - x0) ** 2 + (g["y"] - y0) ** 2)
```

Comment:
Is it bad practice to call this "circ" when you have defined the method `circ` above to mean something else?

Reply:
Variables defined in a function are local, so I don't think it matters much.  Good practice would be variable names that are descriptive.  Can change it. There was something important to catch, that I will need to change anyway...